# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: anaconda 19.31.105\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2014-11-06 17:27-0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"

#: pyanaconda/addons.py:150
#, python-format
msgid "Unhandled arguments on %%addon line for %s"
msgstr ""

#: pyanaconda/bootloader.py:246
msgid "/boot file system"
msgstr ""

#: pyanaconda/bootloader.py:404
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""

#: pyanaconda/bootloader.py:412
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_version)s."
msgstr ""

#: pyanaconda/bootloader.py:420
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:436
#, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:448
#, python-format
msgid "%(desc)s cannot be of type %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:454
#, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr ""

#: pyanaconda/bootloader.py:466
#, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr ""

#: pyanaconda/bootloader.py:471
#, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr ""

#: pyanaconda/bootloader.py:480
#, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr ""

#: pyanaconda/bootloader.py:498
#, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr ""

#: pyanaconda/bootloader.py:508
#, python-format
msgid "%s must be on a primary partition."
msgstr ""

#: pyanaconda/bootloader.py:646 pyanaconda/bootloader.py:747
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr ""

#: pyanaconda/bootloader.py:701
#, python-format
msgid "%s cannot be on an iSCSI disk on s390(x)"
msgstr ""

#: pyanaconda/bootloader.py:705
#, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr ""

#: pyanaconda/bootloader.py:1354
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s in on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1360
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1639
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""

#: pyanaconda/constants.py:85
msgid "Start VNC"
msgstr ""

#: pyanaconda/constants.py:86
msgid "Use text mode"
msgstr ""

#: pyanaconda/constants_text.py:40 pyanaconda/rescue.py:336
#: pyanaconda/rescue.py:368 pyanaconda/rescue.py:381 pyanaconda/rescue.py:449
#: pyanaconda/rescue.py:465 pyanaconda/rescue.py:476 anaconda:451
msgid "OK"
msgstr ""

#: pyanaconda/constants_text.py:44 pyanaconda/text.py:54 pyanaconda/text.py:55
msgid "Cancel"
msgstr ""

#: pyanaconda/constants_text.py:48
msgid "Yes"
msgstr ""

#: pyanaconda/constants_text.py:52
msgid "No"
msgstr ""

#: pyanaconda/errors.py:98
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/errors.py:104
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""

#: pyanaconda/errors.py:110
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""

#: pyanaconda/errors.py:119
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/errors.py:127
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: pyanaconda/errors.py:138
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type:\n"
"\n"
msgstr ""

#: pyanaconda/errors.py:145
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""

#: pyanaconda/errors.py:159
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:170
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr ""

#: pyanaconda/errors.py:179
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/errors.py:190
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:195
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:210
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:215
#, python-format
msgid ""
"You have specified that the package '%s' should be excluded from "
"installation.  This package does not exist.  Would you like to ignore this "
"package and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:228
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:238
#, python-format
msgid ""
"There was an error installing the %s package.  This is a fatal error and "
"installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:242
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:250
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""

#: pyanaconda/errors.py:258
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/errors.py:265
#, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:89
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:93
msgid "Hardware error occured"
msgstr ""

#: pyanaconda/exception.py:125
#, python-format
msgid ""
"\n"
"The installation was stopped due to an error which occurred while running in "
"non-interactive cmdline mode. Since there cannotbe any questions in cmdline "
"mode, edit your kickstart file and retry installation. \n"
"The exact error message is: \n"
"\n"
"%s. \n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/install.py:78
msgid "Configuring installed system"
msgstr ""

#: pyanaconda/install.py:91
msgid "Writing network configuration"
msgstr ""

#: pyanaconda/install.py:95
msgid "Creating users"
msgstr ""

#: pyanaconda/install.py:102
msgid "Configuring addons"
msgstr ""

#: pyanaconda/install.py:105
msgid "Generating initramfs"
msgstr ""

#: pyanaconda/install.py:109
#, python-format
msgid "Joining realm: %s"
msgstr ""

#: pyanaconda/install.py:112
msgid "Running post-installation scripts"
msgstr ""

#: pyanaconda/install.py:159
#, python-format
msgid "Waiting for %s threads to finish"
msgstr ""

#: pyanaconda/install.py:165
msgid "Setting up the installation environment"
msgstr ""

#: pyanaconda/install.py:196
msgid "Discovering realm to join"
msgstr ""

#: pyanaconda/install.py:251
msgid "Installing boot loader"
msgstr ""

#: pyanaconda/install.py:254 pyanaconda/packaging/livepayload.py:157
#: pyanaconda/packaging/yumpayload.py:1642
msgid "Performing post-installation setup tasks"
msgstr ""

#: pyanaconda/kickstart.py:156
#, python-format
msgid "Escrow certificate %s requires network access."
msgstr ""

#: pyanaconda/kickstart.py:164
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/kickstart.py:255
#, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr ""

#: pyanaconda/kickstart.py:280
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr ""

#: pyanaconda/kickstart.py:298
#, python-format
msgid "Settings default fstype to %s failed."
msgstr ""

#: pyanaconda/kickstart.py:653
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr ""

#: pyanaconda/kickstart.py:1154 pyanaconda/kickstart.py:1363
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/kickstart.py:1662
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through redhat-upgrade-tool."
msgstr ""

#: pyanaconda/kickstart.py:1880
msgid "Running pre-installation scripts"
msgstr ""

#: pyanaconda/network.py:97
msgid "Host name cannot be None or an empty string."
msgstr ""

#: pyanaconda/network.py:100
msgid "Host name must be 255 or fewer characters in length."
msgstr ""

#: pyanaconda/network.py:105
msgid ""
"Host name must start with a valid character in the ranges 'a-z', 'A-Z', or "
"'0-9'"
msgstr ""

#: pyanaconda/network.py:113
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1294 pyanaconda/ui/gui/spokes/custom.py:1159
#: pyanaconda/ui/gui/spokes/network.py:299
#: pyanaconda/ui/gui/spokes/lib/accordion.py:59
msgid "Unknown"
msgstr ""

#: pyanaconda/network.py:1298
msgid "Connecting..."
msgstr ""

#: pyanaconda/network.py:1300
msgid "Disconnecting..."
msgstr ""

#: pyanaconda/network.py:1320
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr ""

#: pyanaconda/network.py:1323
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr ""

#: pyanaconda/network.py:1326
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1330
#, python-format
msgid "Team%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1334
#, python-format
msgid "Bridge%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1340
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1357
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1360
msgid "Not connected"
msgstr ""

#: pyanaconda/network.py:1363 pyanaconda/ui/gui/spokes/network.glade.h:45
msgid "No network devices available"
msgstr ""

#: pyanaconda/product.py:53
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr ""

#: pyanaconda/rescue.py:185 pyanaconda/rescue.py:353 pyanaconda/rescue.py:442
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr ""

#: pyanaconda/rescue.py:188
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: pyanaconda/rescue.py:202
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: pyanaconda/rescue.py:291 pyanaconda/rescue.py:364 pyanaconda/rescue.py:374
#: pyanaconda/rescue.py:445 pyanaconda/rescue.py:476
msgid "Rescue"
msgstr ""

#: pyanaconda/rescue.py:292
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  \n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:302
msgid "Continue"
msgstr ""

#: pyanaconda/rescue.py:302 pyanaconda/rescue.py:308
msgid "Read-Only"
msgstr ""

#: pyanaconda/rescue.py:302 pyanaconda/rescue.py:304
msgid "Skip"
msgstr ""

#: pyanaconda/rescue.py:333
msgid "System to Rescue"
msgstr ""

#: pyanaconda/rescue.py:334
msgid "Which device holds the root partition of your installation?"
msgstr ""

#: pyanaconda/rescue.py:336 pyanaconda/rescue.py:340
msgid "Exit"
msgstr ""

#: pyanaconda/rescue.py:350 pyanaconda/rescue.py:439
msgid "The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:365
#, python-format
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions. %s"
msgstr ""

#: pyanaconda/rescue.py:375
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""

#: pyanaconda/rescue.py:446
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell."
msgstr ""

#: pyanaconda/rescue.py:454
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: pyanaconda/rescue.py:458
msgid " The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:462
msgid "Rescue Mode"
msgstr ""

#: pyanaconda/rescue.py:463
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr ""

#: pyanaconda/rescue.py:475
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: pyanaconda/text.py:101
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/text.py:108
msgid "Passphrase"
msgstr ""

#: pyanaconda/users.py:134
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/users.py:140
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: pyanaconda/users.py:147
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""

#: pyanaconda/vnc.py:75
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: pyanaconda/vnc.py:134
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""

#: pyanaconda/vnc.py:153
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr ""

#: pyanaconda/vnc.py:167
msgid "Connected!"
msgstr ""

#: pyanaconda/vnc.py:170
msgid "Will try to connect again in 15 seconds..."
msgstr ""

#: pyanaconda/vnc.py:177
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/vnc.py:188
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""

#: pyanaconda/vnc.py:190
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""

#: pyanaconda/vnc.py:195
msgid "Starting VNC..."
msgstr ""

#: pyanaconda/vnc.py:240
msgid "The VNC server is now running."
msgstr ""

#: pyanaconda/vnc.py:244
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:249
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:253
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:255
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:276
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""

#: pyanaconda/installclasses/fedora.py:30
msgid "_Fedora"
msgstr ""

#: pyanaconda/installclasses/rhel.py:30
msgid "Red Hat Enterprise Linux"
msgstr ""

#: pyanaconda/packaging/__init__.py:808
msgid "Failed to set up installation source"
msgstr ""

#: pyanaconda/packaging/__init__.py:809
msgid "Error downloading package metadata"
msgstr ""

#: pyanaconda/packaging/__init__.py:810
msgid "No installation source available"
msgstr ""

#: pyanaconda/packaging/livepayload.py:91
#: pyanaconda/packaging/livepayload.py:112
msgid "Installing software"
msgstr ""

#: pyanaconda/packaging/livepayload.py:213
#: pyanaconda/packaging/livepayload.py:222
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr ""

#: pyanaconda/packaging/livepayload.py:317
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1389
#, python-format
msgid "CmdlineError: Missing package: %s"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1581
msgid "Starting package installation process"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1622
msgid "You need more space on the following file systems:\n"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1640
msgid "Preparing transaction from installation source"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1641
msgid "Installing"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:157
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:174
#, python-format
msgid "Preparing deployment of %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:186
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:189
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:57
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:66
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:81
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:84
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:88
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/lib/entropy.py:91
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:96
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:99
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/tui/hubs/progress.py:36
msgid "Progress"
msgstr ""

#: pyanaconda/ui/tui/hubs/progress.py:134
#, python-format
msgid ""
"\tUse of this product is subject to the license agreement found at %s\n"
"\n"
"\tInstallation complete.  Press return to quit"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:35
msgid "Installation"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:51
msgid "Starting automated install"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:80
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:93
msgid ""
"  Please make your choice from above ['q' to quit | 'b' to begin "
"installation |\n"
"  'r' to refresh]: "
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/hubs/summary.py:107
#: pyanaconda/ui/tui/spokes/langsupport.py:116
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:110
#: pyanaconda/ui/tui/hubs/__init__.py:112
msgid "Please complete all spokes before continuing"
msgstr ""

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:121
#: pyanaconda/ui/tui/hubs/__init__.py:109
#: pyanaconda/ui/tui/simpleline/base.py:454
msgid "c"
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:42
msgid "Default HUB title"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:29
msgid "Error"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:50
msgid "Press enter to exit."
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:60
msgid "Question"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:83
msgid "Please respond 'yes' or 'no': "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:86
msgid "yes"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:91
msgid "no"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:86
msgid "Do you really want to quit?"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:449
msgid "r"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:459
#: pyanaconda/ui/tui/spokes/askvnc.py:87
msgid "q"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:564
msgid "Press ENTER to continue"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:613
msgid ""
"  Please make your choice from above ['q' to quit | 'c' to continue |\n"
"  'r' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:30
msgid "VNC"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:41
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:99
msgid "VNC Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:108
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:127
#: pyanaconda/ui/gui/spokes/network.py:1324
msgid "Password: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:128
msgid "Password (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:131
msgid "Passwords do not match!"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:134
msgid "The password must be at least six characters long."
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:138
msgid "The password cannot be more than eight characters long."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:39
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:69
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:79
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:82
msgid "Available languages"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:124
msgid ""
"Please select language support to install.\n"
"[b to language list, c to continue, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:40
msgid "Network settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:91
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:96
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:108
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:110
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:117
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:137
#, python-format
msgid "Host name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:149
msgid "Set host name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:151
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:170
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:12
msgid "Host name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:188
msgid ""
"Configuration not saved: gateway or netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:204
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:231
#: pyanaconda/ui/gui/spokes/network.py:1466
#: pyanaconda/ui/gui/spokes/network.py:1537
#, python-format
msgid "Host name is not valid: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:255
msgid "Device configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:259
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:261
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:262
msgid "IPv4 gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:263
#, python-format
msgid ""
"IPv6 address or %(auto)s for automatic, %(dhcp)s for DHCP, %(ignore)s to "
"turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:266
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:267
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:268
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:269
msgid "Apply configuration in installer"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:284
#, python-format
msgid "Configuring device %s."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:31
msgid "Set root password"
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:54
msgid "Password is set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:56
msgid "Root account is disabled."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:58
msgid "Password is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:63
msgid "Please select new root password. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:69
msgid "Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:53
msgid "Use All Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:54
msgid "Replace Existing Linux system(s)"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:55
msgid "Use Free Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:65
msgid "Install Destination"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:118
#: pyanaconda/ui/gui/spokes/storage.py:412
msgid "No disks selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:121
#: pyanaconda/ui/gui/spokes/storage.py:417
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:128
#: pyanaconda/ui/gui/spokes/storage.py:428
msgid "Error checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:130
#: pyanaconda/ui/gui/spokes/storage.py:430
msgid "Warning checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:133
#: pyanaconda/ui/gui/spokes/storage.py:432
msgid "Automatic partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:135
#: pyanaconda/ui/gui/spokes/storage.py:434
msgid "Custom partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:171
#: pyanaconda/ui/gui/spokes/storage.py:678
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:173
#: pyanaconda/ui/gui/spokes/storage.py:680
msgid "No disks selected; please select at least one disk to install to."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:188
#: pyanaconda/ui/tui/spokes/source.py:475
#: pyanaconda/ui/gui/spokes/source.py:569
#: pyanaconda/ui/gui/spokes/storage.py:636
msgid "Probing storage..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:285
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:287
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:302
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:92
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:346
msgid "Generating updated storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:351
#: pyanaconda/ui/tui/spokes/storage.py:363
#, python-format
msgid "storage configuration failed: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:367
#: pyanaconda/ui/gui/spokes/storage.py:246
msgid "Checking storage configuration..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:410
msgid "Autopartitioning Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:441
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:479
msgid "Partition Scheme Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:504
msgid "Select a partition scheme configuration."
msgstr ""

#: pyanaconda/ui/tui/spokes/time.py:29
msgid "Timezone settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time.py:60
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:502
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:506
#, python-format
msgid "%s timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time.py:62
msgid "Timezone is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/time.py:70
#, python-format
msgid "Available timezones in region %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/time.py:73
msgid "Available regions"
msgstr ""

#: pyanaconda/ui/tui/spokes/time.py:136
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [b to region list, c to continue, q to "
"quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:33
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:48
msgid "Start shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:57
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:148
msgid "Installation source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:151
#: pyanaconda/ui/tui/spokes/source.py:209
#: pyanaconda/ui/gui/spokes/source.py:483
#: pyanaconda/ui/gui/spokes/source.glade.h:26
msgid "Closest mirror"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:201
#: pyanaconda/ui/gui/spokes/source.py:475
#, python-format
msgid "NFS server %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:203
#: pyanaconda/ui/gui/spokes/source.py:477
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:206
#: pyanaconda/ui/tui/spokes/source.py:220
#: pyanaconda/ui/gui/spokes/source.py:471
msgid "Error setting up software source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:211
#: pyanaconda/ui/tui/spokes/software.py:84
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:508
#: pyanaconda/ui/gui/spokes/software.py:217
#: pyanaconda/ui/gui/spokes/source.py:485
msgid "Nothing selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:222
#: pyanaconda/ui/tui/spokes/software.py:64
msgid "Processing..."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:228
#, python-format
msgid "Repo URL set to: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:246
msgid "Installation source refresh in progress, try again later."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:252
#: pyanaconda/ui/gui/spokes/source.py:762
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:256
msgid "CD/DVD"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:256
msgid "Local ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:256
msgid "Network"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:260
msgid "Choose an installation source type."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:351
#: pyanaconda/ui/tui/spokes/source.py:389
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:355
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:393
msgid "<server>:/<path>"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:394
msgid "NFS mount options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:428
msgid "Failed to set up installation source. Check the source address."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:437
msgid "Select device containing the ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:496
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:525
msgid "Select an ISO to use as install source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:554
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:37
msgid "Software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:62
#: pyanaconda/ui/gui/spokes/software.py:191
msgid "Error checking software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:66
#: pyanaconda/ui/gui/spokes/software.py:194
msgid "Installation source not set up"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:68
#: pyanaconda/ui/gui/spokes/software.py:197
msgid "Source changed - please verify"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:83
#: pyanaconda/ui/gui/spokes/software.py:208
msgid "Custom software selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:108
msgid "Installation source needs to be set up first."
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:116
msgid "Base environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:36
msgid "Create user"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:113 pyanaconda/ui/gui/spokes/user.py:242
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:115
msgid "You must set a password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:117 pyanaconda/ui/gui/spokes/user.py:244
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:119 pyanaconda/ui/gui/spokes/user.py:246
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings.py:36
msgid "Warnings"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings.py:44
#: pyanaconda/ui/gui/spokes/welcome.glade.h:4
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:48
msgid "Default spoke title"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:58
msgid "testing status..."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:98
msgid "New value"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:115
#, python-format
msgid "%s: "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:116
#, python-format
msgid "%s (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:127
#: pyanaconda/ui/gui/spokes/password.py:126
#: pyanaconda/ui/gui/spokes/user.py:360
msgid "The password you have provided is weak"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:131
#, python-format
msgid "You have provided a weak password: %s. "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:132
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:141
#, python-format
msgid "Enter new value for '%s' and press enter\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:149
#, python-format
msgid ""
"You have provided an invalid user name: %s\n"
"Tip: Keep your user name shorter than 32 characters and do not use spaces.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:237
msgid "Password set."
msgstr ""

#: pyanaconda/ui/gui/__init__.py:512 pyanaconda/ui/gui/spokes/software.py:417
#: pyanaconda/ui/gui/spokes/storage.py:977
#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
#: widgets/src/StandaloneWindow.c:54
#: pyanaconda/ui/gui/spokes/welcome.glade.h:1
#: pyanaconda/ui/gui/hubs/summary.glade.h:2
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:527 pyanaconda/ui/gui/main.glade.h:2
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:527 pyanaconda/ui/gui/main.glade.h:3
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/categories/__init__.py:47
msgid "DEFAULT TITLE"
msgstr ""

#: pyanaconda/ui/gui/categories/customization.py:32
msgid "CUSTOMIZATION"
msgstr ""

#: pyanaconda/ui/gui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr ""

#: pyanaconda/ui/gui/categories/software.py:31
msgid "SOFTWARE"
msgstr ""

#: pyanaconda/ui/gui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:160
msgid "SYSTEM"
msgstr ""

#: pyanaconda/ui/gui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr ""

#: pyanaconda/ui/gui/categories/licensing.py:31
msgid "LICENSING"
msgstr ""

#: pyanaconda/ui/gui/hubs/__init__.py:303
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:111
msgid "Complete!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:135
#, python-format
msgid "Use of this product is subject to the license agreement found at %s"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:208
#, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:104
#, python-format
msgid "New %(name)s %(version)s Installation"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:105
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:106
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:107
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:108
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:110
msgid "Volume Group"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:111
msgid "Volume"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:112
msgid ""
"Storage configuration reset due to unrecoverable error. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:114
msgid "Device reconfiguration failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:117
msgid "Unacceptable label format for file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:118
msgid "Cannot set label on file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:119
msgid "Cannot relabel already existing file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:121
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:122
msgid "That mount point is invalid. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:123
msgid "That mount point is already in use. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:125
#, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:128
#: pyanaconda/ui/gui/spokes/custom.glade.h:38
msgid "Please enter a valid name."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:129
msgid "That name is invalid. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:157
#: pyanaconda/ui/gui/spokes/lib/accordion.py:324
#: pyanaconda/ui/gui/spokes/custom.glade.h:49
msgid "LVM"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:158
#: pyanaconda/ui/gui/spokes/lib/accordion.py:325
#: pyanaconda/ui/gui/spokes/custom.glade.h:51
msgid "LVM Thin Provisioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:159
#: pyanaconda/ui/gui/spokes/custom.glade.h:50
msgid "RAID"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:160
#: pyanaconda/ui/gui/spokes/lib/accordion.py:322
#: pyanaconda/ui/gui/spokes/custom.glade.h:47
msgid "Standard Partition"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:161
#: pyanaconda/ui/gui/spokes/lib/accordion.py:323
#: pyanaconda/ui/gui/spokes/custom.glade.h:48
msgid "Btrfs"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:162
#: pyanaconda/ui/gui/spokes/custom.glade.h:10
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:4
msgid "Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:503
#: pyanaconda/ui/gui/spokes/custom.glade.h:2
#, c-format, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:505
#, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:595
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:793
#: pyanaconda/ui/gui/spokes/custom.glade.h:43
msgid "MANUAL PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:983
#: pyanaconda/ui/gui/spokes/filter.py:614
#, python-format
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom.py:1109
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1182
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1185
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1188
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1191
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1194
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1197
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1399
#, python-format
msgid "/boot/efi must be on a device of type %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1403
#, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1406
#: pyanaconda/ui/gui/spokes/custom.py:1408
#, python-format
msgid "%s cannot be encrypted"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1410
msgid "You must create a new file system on the root device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1415
#, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1662
msgid "Device resize request failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1738
msgid "Device reformat request failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1791
#, python-format
msgid "Specified name %s already in use."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1869
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1874
#, python-format
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom.py:1959
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2113
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2115
msgid "This file system may not be resized."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2169
msgid ""
"Error checking storage configuration.  Click for details or press Done again "
"to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2171
msgid ""
"Warning checking storage configuration.  Click for details or press Done "
"again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2345
#, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2357
msgid "Failed to add new device. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2363
msgid "Invalid partition size set. Use a valid integer."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2394
msgid "Device removal request failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2633
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2664
#, python-format
msgid "(%s free)"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2805
#, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2811
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2886
msgid "No disks selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2892
msgid "Not enough free space on selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2898
#: pyanaconda/ui/gui/spokes/custom.py:2924
msgid "Automatic partitioning failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:3189
msgid "Failed to unlock encrypted block device. Click for details"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:3214
#, python-format
msgid ""
"You have chosen to manually set up the file systems for your new "
"%(productName)s installation. Before you begin, you might want to take a "
"minute to learn the lay of the land. Quite a bit has changed.\n"
"\n"
"The most important change is that creation of new file systems has been "
"streamlined. You no longer have to build complex devices like LVM logical "
"volumes in stages (physical volume, then volume group, then logical volume) "
"-- now you just create a logical volume and we'll handle the legwork of "
"setting up the physical volumes and volume group to contain it. We'll also "
"handle adjusting the volume group as you add, remove, and resize logical "
"volumes so you don't have to worry about the mundane details.\n"
"\n"
"\n"
"Screen Layout\n"
"\n"
"The left-hand side of the screen shows the OS installations we were able to "
"find on this computer. The new %(productName)s installation is at the top of "
"the list. You can click on the names of the installations to see what file "
"systems they contain.\n"
"\n"
"Below the various installations and mount points on the left-hand side there "
"are buttons to add a new file system, remove the selected file system, or "
"configure the selected file system.\n"
"\n"
"The right-hand side of the screen is where you can customize the currently-"
"selected mount point.\n"
"\n"
"On the bottom-left you will see a summary of the disks you have chosen to "
"use for the installation. You can click on the blue text to see more "
"detailed information about your selected disks.\n"
"\n"
"\n"
"How to create a new file system on a new device\n"
"\n"
"1. Click on the + button.\n"
"2. Enter the mount point and size. (Hint: Hover the mouse pointer over "
"either of the text entry areas for help.)\n"
"3. Select the new mount point under \"New %(productName)s Installation\" on "
"the left-hand side of the screen and customize it to suit your needs.\n"
"\n"
"\n"
"How to reformat a device/file system that already exists on your disk\n"
"\n"
"1. Select the file system from the left-hand side of the screen.\n"
"2. Click on the \"Customize\" expander in the mount point customization area "
"on the right-hand side of the screen.\n"
"3. Activate the \"Reformat\" checkbutton, select a file system type and, if "
"applicable, enter a mount point above in the \"Mount point\" text entry "
"area.\n"
"4. Click on \"Apply changes\"\n"
"\n"
"\n"
"How to set a mount point for a file system that already exists on your disk\n"
"\n"
"1. Select the file system from the left-hand side of the screen.\n"
"2. Enter a mount point in the \"Mount point\" text entry area in the mount "
"point customization area.\n"
"3. Click on \"Apply changes\"\n"
"\n"
"\n"
"How to remove a file system that already exists on your disk\n"
"\n"
"1. Select the file system you wish to remove on the left-hand side of the "
"screen.\n"
"2. Click the - button.\n"
"\n"
"Hint: Removing a device that already exists on your disk from the \"New "
"%(productName)s Installation\" does not remove it from the disk. It only "
"resets that device to its original state. To remove a device that already "
"exists on your disk, you must select it from under any of the other detected "
"installations (or \"Unknown\") and hit the - button.\n"
"\n"
"\n"
"Tips and hints\n"
"\n"
"You can enter sizes for new file systems that are greater than the total "
"available free space. The installer will come as close as possible to the "
"size you request.\n"
"\n"
"By default, new devices use any/all of your selected disks.\n"
"\n"
"You can change which disks a new device may be allocated from by clicking "
"the configure button (the one with a tools graphic) while that device is "
"selected.\n"
"\n"
"When adding a new mount point by clicking the + button, leave the size entry "
"blank to make the new device use all available free space.\n"
"\n"
"When you remove the last device from a container device like an LVM volume "
"group, we will automatically remove that container device to make room for "
"new devices.\n"
"\n"
"When the last partition is removed from a disk, that disk may be "
"reinitialized with a new partition table if we think there is a more "
"appropriate type for that disk.\n"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:60
msgid "Note: pool servers may not be available all the time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:376
msgid "DATE & _TIME"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:493
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:504
msgid "Invalid timezone"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:674
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:688
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:872
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:880
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:8
msgid "PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:676
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:685
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:871
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:874
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:879
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:882
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1014
msgid "You need to set up networking first if you want to use NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1019
msgid "You have no working NTP server configured"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:508
msgid "_INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:39
#, python-format
msgid "%s to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:263
msgid "_KEYBOARD"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:330
msgid "Testing layouts configuration not available."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:382
msgid "Layout switching not configured."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.py:54
msgid "_LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:85
msgid "Status unknown (missing)"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:88
msgid "Status unknown"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:90
msgid "Unmanaged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:93
#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Unavailable"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:95
msgid "Firmware missing"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:101
msgid "Cable unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:103
msgid "Disconnected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:108
msgid "Connecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Authentication required"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Connected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:114
msgid "Disconnecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:116
msgid "Connection failed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:300
msgid "Ethernet"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:301
msgid "Wireless"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:302
#: pyanaconda/ui/gui/spokes/network.glade.h:3
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:303
#: pyanaconda/ui/gui/spokes/network.glade.h:7
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:304
#: pyanaconda/ui/gui/spokes/network.glade.h:6
msgid "Team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:305
#: pyanaconda/ui/gui/spokes/network.glade.h:4
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:731
msgid "unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:820
#: pyanaconda/ui/gui/spokes/network.glade.h:18
msgid "IPv4 Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:821
#: pyanaconda/ui/gui/spokes/network.glade.h:17
msgid "IPv6 Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:823
#: pyanaconda/ui/gui/spokes/network.py:825
#: pyanaconda/ui/gui/spokes/network.glade.h:33
msgid "IP Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:915
#, python-format
msgid "%d Mb/s"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1135
msgid "WEP"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1138
msgid "WPA"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1141
msgid "WPA2"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1145
msgid "Enterprise"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1150 widgets/src/SpokeSelector.c:55
#: pyanaconda/ui/gui/spokes/filter.glade.h:3
#: pyanaconda/ui/gui/spokes/custom.glade.h:24
msgid "None"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1286
#: pyanaconda/ui/gui/spokes/network.glade.h:50
msgid "Authentication required by wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1287
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1301
msgid "_Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1309
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1319
msgid "User name: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1330
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1335
msgid "Private key password: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1390
msgid "_NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:45
msgid "_ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:62
#: pyanaconda/ui/gui/spokes/password.py:63
msgid "The password is set."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:75
msgid "Error setting root password"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:77
msgid "Root password is set"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:79
msgid "Root account is disabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:81
msgid "Root password is not set"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:125
#: pyanaconda/ui/gui/spokes/user.py:359
#: pyanaconda/ui/gui/spokes/lib/passphrase.py:111
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:6
msgid "Weak"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:131
#: pyanaconda/ui/gui/spokes/user.py:365
msgid "You will have to press Done twice to confirm it."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:134
#: pyanaconda/ui/gui/spokes/user.py:368
#: pyanaconda/ui/gui/spokes/lib/passphrase.py:114
msgid "Fair"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:138
#: pyanaconda/ui/gui/spokes/user.py:372
#: pyanaconda/ui/gui/spokes/lib/passphrase.py:117
msgid "Good"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:142
#: pyanaconda/ui/gui/spokes/user.py:376
#: pyanaconda/ui/gui/spokes/lib/passphrase.py:120
msgid "Strong"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:147
#: pyanaconda/ui/gui/spokes/user.py:381
msgid "Empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:148
#: pyanaconda/ui/gui/spokes/user.py:382
msgid "The password is empty."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:150
#: pyanaconda/ui/gui/spokes/user.py:384
msgid "The passwords do not match."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:175
#: pyanaconda/ui/gui/spokes/user.py:418
msgid "You must provide and confirm a password."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:185
#: pyanaconda/ui/gui/spokes/user.py:428
#, python-format
msgid "You have provided a weak password: %s.  Press Done again to use anyway."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:49
msgid "_SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:81
#: pyanaconda/ui/gui/spokes/source.py:52
msgid "Downloading package metadata..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:84
msgid "Downloading group metadata..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:121
#: pyanaconda/ui/gui/spokes/source.py:465
msgid "Checking software dependencies..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:126
msgid "Error checking software dependencies"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:214
msgid "Unknown custom software selected"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:336
msgid "Error checking software dependencies.  Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:414
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source.  You can change your installation source or quit "
"the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:417
#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
#: pyanaconda/ui/gui/spokes/storage.glade.h:11
#: pyanaconda/ui/gui/spokes/source.glade.h:4
#: pyanaconda/ui/gui/spokes/custom.glade.h:5
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:1
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:9
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:418
msgid "_Modify Software Source"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:51
msgid "Setting up installation source..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:168
msgid "This media is good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:170
msgid "This media is not good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:292
msgid "_INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:405
#: pyanaconda/ui/gui/spokes/source.py:585
msgid "Failed to set up installation source; check the repo url"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:469
msgid "Error setting up base repository"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:480
msgid "Error setting up ISO file"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:587
msgid ""
"Failed to set up installation source; check the repo url and proxy settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:612
#, python-format
msgid "Device: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:613
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:774
msgid "You need to configure the network to use a network installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1046
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1057
msgid ""
"Duplicate repository names not allowed; choose a unique name for each "
"repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1079
msgid "Empty repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1083
msgid "Invalid repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1086
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:112
#, python-format
msgid ""
"Your current <a href=\"\"><b>%(product)s</b> software selection</a> requires "
"<b>%(total)s</b> of available space, including <b>%(software)s</b> for "
"software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:163
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:167
#: pyanaconda/ui/gui/spokes/storage.py:205
msgid "Please wait... software metadata still loading."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:174
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:198
#, python-format
msgid ""
"%(sw_text)s You don't have enough space available to install <b>%(product)s</"
"b>, even if you used all of the free space available on the selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:212
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:267
msgid "INSTALLATION _DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:363
msgid "Saving storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:369
#: pyanaconda/ui/gui/spokes/storage.py:381
msgid "Failed to save storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:415
msgid "Not configured"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:419
#: pyanaconda/ui/gui/spokes/storage.py:711
msgid "Formatting DASDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:563
msgid "Error checking storage configuration.  Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:565
msgid "Warning checking storage configuration.  Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:610
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:620
#, python-format
msgid "%s free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:665
#, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/storage.py:748
msgid ""
"You have chosen to skip boot loader installation.  Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:957
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:973
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:977
msgid "_Modify Storage Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:990
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:994
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:2
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:10
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:149
msgid "_USER CREATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:212 pyanaconda/ui/gui/spokes/user.py:213
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:240
#, python-format
msgid "Error creating user account: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:474
msgid "Invalid user name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:155
msgid "DATA"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:262
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:274
msgid "_Click here to create them automatically."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:297
msgid "Create new mount points by clicking the '+' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:309
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:315
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:133
#, python-format
msgid ""
"<b>%(count)s disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:191
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:8
msgid "_Set as Boot Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:32
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:33
msgid "Passphrases do not match."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:54
msgid "Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:55
msgid "Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:56
msgid "Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:115
msgid "This device contains the installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:161
#, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:167
msgid "Not resizeable"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:191
msgid "<span foreground='grey' style='italic'>Free space</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:211
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:217
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:227
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:234
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:436
#: pyanaconda/ui/gui/spokes/lib/resize.py:441
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:12
msgid "Delete _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:438
msgid "Preserve _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:52
#, python-format
msgid "%(description)s (%(deviceName)s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:56
#, python-format
msgid "%(deviceName)s on %(container)s"
msgstr ""

#: widgets/src/BaseWindow.c:89
msgid "DISTRIBUTION INSTALLATION"
msgstr ""

#: widgets/src/BaseWindow.c:90
msgid "SPOKE NAME"
msgstr ""

#: widgets/src/BaseWindow.c:91
msgid "PRE-RELEASE / TESTING"
msgstr ""

#: widgets/src/BaseWindow.c:93
msgid "Help!"
msgstr ""

#: widgets/src/DiskOverview.c:52
msgid "New Device"
msgstr ""

#: widgets/src/DiskOverview.c:54 widgets/src/DiskOverview.c:55
msgid "0 MB"
msgstr ""

#: widgets/src/MountpointSelector.c:49
msgid "0 GB"
msgstr ""

#: widgets/src/SpokeSelector.c:56
msgid "New Selector"
msgstr ""

#: widgets/src/SpokeWindow.c:48 pyanaconda/ui/gui/spokes/source.glade.h:1
msgid "_Done"
msgstr ""

#: widgets/src/StandaloneWindow.c:55
#: pyanaconda/ui/gui/spokes/welcome.glade.h:2
msgid "_Continue"
msgstr ""

#: widgets/src/LayoutIndicator.c:31
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:32
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:1
msgid "_Save Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:2
msgid "ADVANCED USER CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:3
msgid "Home Directory"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:4
msgid "Create a _home directory for this user."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:5
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:6
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:7
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:8
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:9
msgid "Group Membership"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:10
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:11
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:12
msgid ""
"You may input a comma-separated list of group names and group IDs here.\n"
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:14
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:15
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:1
msgid "DATE & TIME"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:2
msgid "_Region:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:3
msgid "_City:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:4
msgid "_Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:5
msgid "00"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:6
msgid ":"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:7
msgid " "
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:9
msgid "24-_hour"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:10
msgid "_AM/PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:11
msgid "Add and mark for usage NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:13
msgid "Working"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:14
msgid "Use"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:1
#: pyanaconda/ui/gui/spokes/storage.glade.h:12
msgid "INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:2
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:4
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:5
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:6
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:7
msgid "_Target:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:8
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:9
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:10
msgid "Search Results:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:11
#: pyanaconda/ui/gui/spokes/keyboard.glade.h:3
#: pyanaconda/ui/gui/spokes/source.glade.h:35
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:5
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:5
msgid "Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:12
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:13
#: pyanaconda/ui/gui/spokes/custom.glade.h:11
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:6
msgid "Capacity"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:14
msgid "Interconnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:15
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:16
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:17
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:18
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:19
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:20
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:21
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:22
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:23
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:24
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:25
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:26
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:27
msgid "_Filter By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:28
#: pyanaconda/ui/gui/spokes/custom.glade.h:13
msgid "ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:29
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:30
msgid "_Other SAN Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:31
msgid "Nothing here yet."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:32
msgid "Firmware _RAID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:33
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:34
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:35
msgid "_CCW:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:36
msgid "_WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:37
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:38
#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:7
msgid "Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:39
msgid "System _z Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:40
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:41
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:42
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:43
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:44
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:46
#: pyanaconda/ui/gui/spokes/custom.glade.h:79
#, c-format, python-format
msgid "%d _storage devices selected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:1
msgid "Add device"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:2
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:5
msgid "bridge"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:8
msgid "NETWORK CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:9
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:11
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:12
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:13
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:14
msgid "DNS"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:15
msgid "Default Route"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:16
msgid "Subnet Mask"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:19
msgid "Speed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:20
msgid "Hardware Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:21
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:22
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:23
msgid "C_onfigure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:24
msgid "Security"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:25
msgid "_Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:26
msgid "Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:27
msgid "Security Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:28
msgid "_Use as Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:29
msgid "_Stop Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:30
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:31
msgid "IMEI"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:32
msgid "Provider"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:34
msgid "VPN Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:35
msgid "Gateway"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:36
msgid "Group Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:37
msgid "Group Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:38
msgid "User name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:39
msgid "_Method"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:40
msgid "_Configuration URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:41
msgid "HTTP _Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:42
msgid "H_TTPS Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:43
msgid "_FTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:44
msgid "_Socks Host"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:46
msgid "Unloc_k"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:47
msgid "_Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:48
msgid "_Host name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:49
msgid "Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:51
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade.h:1
msgid "SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade.h:2
msgid "Base Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade.h:3
msgid "Add-Ons for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade.h:4
msgid "column"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:1
msgid "Cancel & _add more disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:2
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:1
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:3
msgid "INSTALLATION OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:4
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:5
msgid "disk free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:6
msgid "Free space available for use."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:7
msgid "fs free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:8
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:9
msgid "Here we'll describe what your options are."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:10
msgid "_Quit installer"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:13
msgid "Device Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:14
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:15
msgid "Local Standard Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:16
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:17
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:18
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:19
msgid "Other Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:20
msgid "Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:21
msgid "Encryption"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:22
msgid "A_utomatically configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:23
msgid "_I will configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:24
msgid "_Encrypt my data."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:25
msgid "You'll set a passphrase later."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:26
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:28
msgid "summary"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:1
msgid "ADD A KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:2
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:4
msgid "KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:5
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:6
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:7
msgid "name"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:8
msgid "_Add layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:9
msgid "_Remove layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:10
msgid "Move selected layout _up"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:11
msgid "Move selected layout _down"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:12
msgid "_Preview layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:13
msgid "_Test the layout configuration below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:14
msgid "Alt + Shift to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:15
msgid "_Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:16
msgid "LAYOUT SWITCHING OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:17
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:18
msgid "use"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:19
msgid "description"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:1
msgid "LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:2
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:3
#: pyanaconda/ui/gui/spokes/welcome.glade.h:16
msgid "nativeName"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:4
#: pyanaconda/ui/gui/spokes/welcome.glade.h:17
msgid "englishName"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:5
#: pyanaconda/ui/gui/spokes/welcome.glade.h:18
msgid "selected"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:6
#: pyanaconda/ui/gui/spokes/welcome.glade.h:19
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:7
msgid "checked"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade.h:1
msgid "ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade.h:2
msgid "_Root Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade.h:3
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:5
msgid "_Confirm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade.h:4
#: pyanaconda/ui/gui/spokes/user.glade.h:8
msgid "empty password"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade.h:5
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:2
msgid "MEDIA VERIFICATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:3
msgid "Verifying media, please wait..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:5
#: pyanaconda/ui/gui/spokes/custom.glade.h:68
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:6
msgid "_Enable HTTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:7
msgid "_Proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:8
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:9
msgid "_Use Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:10
msgid "User _name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:11
msgid "Pass_word"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:12
msgid "INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:13
msgid "Which installation source would you like to use?"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:14
msgid "_Auto-detected installation media:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:15
msgid "Device:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:16
msgid "Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:17
msgid "_Verify"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:18
msgid "_ISO file:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:19
msgid "D_evice:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:20
msgid "_Choose an ISO"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:21
msgid "_On the network:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:22
msgid "http://"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:23
msgid "https://"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:24
msgid "ftp://"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:25
msgid "nfs"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:27
msgid "_Proxy setup..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:28
msgid "This URL refers to a _mirror list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:29
msgid "N_FS mount options:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:30
msgid "This field is optional."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:31
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:32
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the install source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:33
msgid "Additional repositories"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:34
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:36
msgid "Add a new repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:37
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:38
msgid "Remove the selected repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:39
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:9
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:40
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:41
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:42
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:43
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:44
msgid "Optional proxy user name."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:45
msgid "Optional proxy password."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:46
msgid "This URL refers to a mirror _list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:47
#: pyanaconda/ui/gui/spokes/custom.glade.h:35
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:48
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:49
msgid "file://"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:50
msgid "Pro_xy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:51
msgid "U_ser Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:52
msgid "Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:53
msgid "Invalid repository name. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:54
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:55
msgid "Duplicate names in list of additional repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:3
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:5
msgid "I want to _exit."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:6
msgid "I want to _proceed."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:7
msgid "This is unstable, pre-release software."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:8
msgid ""
"Notice:  This is pre-release software that is intended for development and "
"testing purposes only.  Do not use this software for any critical work or "
"for production environments.\n"
"\n"
"By clicking \"I want to proceed\", you understand and accept the risks "
"associated with pre-release software, that you intend to use this for "
"testing and development purposes only, and are willing to report any bugs or "
"issues in order to enhance this work.\n"
"\n"
"If you do not understand or accept the risks, then please exit this program "
"by clicking \"I want to exit\" which will reboot your system."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:14
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:15
msgid "What language would you like to use during the installation process?"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:1
msgid "CREATE USER"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:2
msgid "_Full name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:3
msgid "_User name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:4
msgid "_Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:5
msgid "_Confirm password"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:6
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:7
msgid "Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:9
msgid "Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:10
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:4
#, c-format, python-format
msgid "Delete _all other file systems in the %s root as well."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:6
msgid "_Delete It"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:7
msgid "_Select"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:8
msgid "CONFIGURE MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:9
msgid "Select one or more disks this device may reside on."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:12
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:7
msgid "Free"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:14
msgid "Help"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:15
msgid "_Add mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:16
msgid "_Desired Capacity:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:17
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:18
msgid "ADD A NEW MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:19
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:21
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:23
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:25
msgid "Single <span foreground=\"grey\">(No Redundancy, No Striping)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:26
msgid "RAID0 <span foreground=\"grey\">(Performance)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:27
msgid "RAID1 <span foreground=\"grey\">(Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:28
msgid "RAID4 <span foreground=\"grey\">(Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:29
msgid "RAID5 <span foreground=\"grey\">(Distributed Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:30
msgid "RAID6 <span foreground=\"grey\">(Redundant Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:31
msgid "RAID10 <span foreground=\"grey\">(Performance, Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:32
msgid "_Save"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:33
msgid "CONFIGURE CONTAINER"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:34
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:36
msgid "RAID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:37
msgid "Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:39
msgid "Si_ze policy:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:40
msgid "Automatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:41
msgid "As large as possible"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:42
msgid "Fixed"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:44
msgid "Selected Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:45
msgid "Device description"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:46
msgid "Device _Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:52
msgid "_Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:53
msgid "File S_ystem:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:54
msgid "Ref_ormat"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:55
msgid "Volume _Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:57
msgid "RA_ID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:58
msgid "_Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:59
msgid "Device(s):"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:61
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:62
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:63
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:64
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:4
msgid "_Passphrase:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:65
msgid "_Unlock"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:66
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:67
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:69
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:70
msgid "Remo_ve"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:71
msgid "Reload storage configuration from disk."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:72
msgid "Re_fresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:73
msgid "Click for help."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:74
msgid "_Help"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:75
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:76
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:77
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:1
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:2
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:3
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:4
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:5
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:6
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:7
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:1
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:2
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:3
msgid "_Target IP Address:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:4
msgid "iSCSI _Initiator Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:5
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:6
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:7
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:8
msgid "_Discovery Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:9
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:10
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:11
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:12
msgid "Reverse CHAP User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:13
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:14
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:5
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:6
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:15
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:16
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:17
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:18
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:9
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:10
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:19
msgid "_Bind targets to network interfaces"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:21
#, python-format
msgid ""
"The following nodes have been discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:22
msgid "Node Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:23
msgid "Interface"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:24
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:25
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:26
msgid "_Reverse CHAP User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:27
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:28
msgid "Node login failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:29
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:30
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:1
msgid "ADD zFCP STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:2
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:3
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:5
msgid "_Device number:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:4
msgid "WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:6
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:7
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:8
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:8
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:10
msgid "LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:3
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:4
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:7
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:9
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:1
msgid "SELECTED DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:2
msgid "_Close"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:3
msgid "Boot"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:4
msgid "Description"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:10
msgid "Disk summary goes here"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:1
msgid "UNFORMATTED DASDS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:2
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:3
msgid "Unformatted DASDs Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:4
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:5
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:6
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:8
msgid "Disk formatting complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade.h:1
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:1
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:2
msgid "_Save Passphrase"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:3
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:7
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:1
msgid "RESCAN DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:2
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:3
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:4
msgid "_Rescan Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:5
msgid "page 1"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:6
msgid "Scanning disks.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:7
msgid "page 2"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:8
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:10
msgid "page 3"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:11
msgid "Disk rescan complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:12
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:13
msgid "page 4"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:2
msgid "RECLAIM DISK SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:3
msgid "Description goes here."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:6
msgid "File System"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:7
msgid "Reclaimable Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:8
#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:6
msgid "Action"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:9
msgid "_Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:10
msgid "_Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:11
msgid "_Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:14
#, c-format, python-format
msgid "Total selected space to reclaim:  <b>%s</b>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:16
#, c-format, python-format
msgid "Installation requires a total of <b>%s</b> for system data."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:1
msgid "SUMMARY OF CHANGES"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:2
msgid "_Cancel & Return to Custom Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:3
msgid "_Accept Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:4
msgid ""
"Your customizations will result in the following changes taking effect on "
"the disks you've selected:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:5
msgid "Order"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:8
msgid "Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:9
msgid "Mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade.h:1
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade.h:2
msgid "Random data quality:"
msgstr ""

#: pyanaconda/ui/gui/main.glade.h:1
msgid "_Exit Installer"
msgstr ""

#: pyanaconda/ui/gui/main.glade.h:4
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade.h:1
msgid "INSTALLATION SUMMARY"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade.h:3
msgid "_Begin Installation"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade.h:4
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade.h:1
msgid "CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade.h:2
msgid "Preparing to install"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade.h:4
#, c-format, python-format
msgid ""
"%s is now successfully installed on your system, but some configuration "
"still needs to be done.  Finish it and then click the Finish configuration "
"button please."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade.h:5
msgid "_Finish configuration"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade.h:7
#, c-format, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"Go ahead and reboot to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade.h:8
msgid "_Reboot"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr ""

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr ""

#: data/liveinst/gnome/fedora-welcome.desktop.in.h:1
#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/gnome/fedora-welcome:64
msgid "Welcome to Fedora"
msgstr ""

#: anaconda:406
#, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""

#: anaconda:408
#, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB."
msgstr ""

#: anaconda:411
msgid ""
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: anaconda:413
msgid "Not enough RAM"
msgstr ""

#: anaconda:414
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""

#: anaconda:417
msgid " Starting text mode."
msgstr ""

#: anaconda:449
msgid "Fatal Error"
msgstr ""

#: anaconda:573
msgid "DISPLAY variable not set. Starting text mode."
msgstr ""

#: anaconda:581
msgid "Graphical installation is not available. Starting text mode."
msgstr ""

#: anaconda:589
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""

#: anaconda:712
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr ""

#: anaconda:714
msgid "Please ssh install@<host> to continue installation."
msgstr ""

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/gnome/fedora-welcome:93
msgid "Try Fedora"
msgstr ""

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/gnome/fedora-welcome:111
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/gnome/fedora-welcome:129
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/gnome/fedora-welcome:132
msgid "Close"
msgstr ""
